docs/pages/user-guides.mdx

```diff
@@ -0,0 +1,224 @@
+---
+title: User Guides
+description: Documentation for end users
+h1: End User Documentation
+---
+
+Teleport is a complex security solution, and most of our documentation covers
+the installation, configuration, and management of Teleport for administrators.
+For end users, this page covers the basics of authenticating to Teleport and
+accessing resources.
+
+## tsh
```

Comment:
What would you think of making this H2 about "Client tools", rather than the first two H2s about client tools and the next few about protocols? To make the structure clearer, we could also have one H2 about tools and another H2 about protocols, with each specific protocol having its own H3.

Reply:
I like that idea conceptually. I went back and forth on nesting headers similar to how you suggest when writing. The factor that made me go this way was that our ToC doesn't render H3s, and I wanted new end users to be able to easily see the thing they were looking for there.

I don't suppose we have a frontmatter value or something that lets us adjust the depth of ToC'd headers, is there?

Reply:
Adjusted headers so that the top-level ToC is broken down into "Clients" and "Protocols"